src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1010,6 +1010,22 @@ public Response createNewBuiltinRole(RoleDTO roleDto) {
 			actionLogSvc.log(alr);
```

Comment:
Above, at `return ok(json(rolesSvc.save(roleDto.asRole())));` I'm getting the following error when trying to set up Dataverse for the first time:

`{"status":"ERROR","message":"Exception thrown from bean: jakarta.ejb.EJBTransactionRolledbackException: Exception thrown from bean: java.lang.NullPointerException: Cannot invoke \"edu.harvard.iq.dataverse.DvObject.isInstanceofDataverse()\" because \"definitionPoint\" is null"}`

Specifically, I'm doing this:

```
rm -rf docker-dev-volumes

mvn -Pct clean package docker:run
```

@luddaniel can you replicate this?

Reply:
`setup-all.sh` creates roles before creating the root dataverse. Causing issue while creating a role 
```
owner = dataverseService.findByAlias("root");
IndexResponse indexDefinitionPointResult = indexDefinitionPoint(owner);
```
Resulting in `"definitionPoint" is null"`

Fixed in my last commit.

Reply:
I repeated this test and it works now:

```
rm -rf docker-dev-volumes

mvn -Pct clean package docker:run
```